### DISCUSSION TITLE
[Orca Slicer] Optimal Vase Mode settings to balance speed and quality?

### DISCUSSION THREAD

**Discussion by Ltek** [OrcaSlicer/OrcaSlicer#3057] [Q&A]
The question... What specific settings are you changing/using to get speed while keeping quality?

I cant get vase prints to print quickly (> 100mms) even setting cooling -> min layer time to 3 seconds. 
Its going slow (20-30mms) on small diameters and no more than about 80mms on larger diameters. It prints far faster on regular (non-vase mode) prints.

Xmas bobble and tree printing time so hoping this helps many.

**Comment by otede** [comment]
Bump!

An example of SANE DEFAULTS RULE broken. In my case printing a 25 mm pipe in Vase mode gets me to about 1 mm cubic flow. It should be 15!

A proposed solution:
In case of Vase mode treat the Filament > Minimum Layer Time as 0 ; then only the Filament > Maximum Flow is the limiting factor.
